assignments/prototype-refactor.js

```diff
@@ -1,9 +1,155 @@
-/* 
+// /* 
 
-Prototype Refactor
+// Prototype Refactor
 
-1. Copy and paste your code or the solution from yesterday
+// 1. Copy and paste your code or the solution from yesterday
 
-2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
+// 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
-*/
+// */
+// /*
+//   EXAMPLE TASK:
+//     - Write an Airplane constructor that initializes `name` from an argument.
+//     - All airplanes built with Airplane should initialize with an `isFlying` of false.
+//     - Give airplanes the ability to `.takeOff()` and `.land()`:
+//         + If a plane takes off, its `isFlying` property is set to true.
+//         + If a plane lands, its `isFlying` property is set to false.
+// */
+
+// // EXAMPLE SOLUTION CODE:
+// function Airplane(name) {
+//   this.name = name;
+//   this.isFlying = false;
+// }
+// Airplane.prototype.takeOff = function() {
+//   this.isFlying = true;
+// };
+// Airplane.prototype.land = function() {
+//   this.isFlying = false;
+// };
+
+// /*
+//   // 👇 COMPLETE YOUR WORK BELOW 👇
+//   // 👇 COMPLETE YOUR WORK BELOW 👇
+//   // 👇 COMPLETE YOUR WORK BELOW 👇
+//   */
+
+// /*
+//     TASK 1
+//       - Write a Person Constructor that initializes `name` and `age` from arguments.
+//       - All instances of Person should initialize with an empty `stomach` array.
+//       - Give instances of Person the ability to `.eat("someFood")`:
+//           + When eating an edible, it should be pushed into the `stomach`.
+//           + The `eat` method should have no effect if there are 10 items in the `stomach`.
+//       - Give instances of Person the ability to `.poop()`:
+//           + When an instance poops, its `stomach` should empty.
+//       - Give instances of Person a method `.toString()`:
+//           + It should return a string with `name` and `age`. Example: "Mary, 50"
+//   */
+
+// class Person {
```

Comment:
Good job on the refactor here too. I noticed you commented out the code given the Person class here was conflicting with the Person class in your other file. In these cases, you can always assign a more descriptive name i.e. `PersonPrototype` vs `PersonClasses`.